src/utils/cesium-tileset-downloader.ts

```diff
@@ -58,9 +97,10 @@ export async function listTilesetUrls(
 ): Promise<string[]> {
   const {signal, foundUrls} = options;
   if (node.boundingVolume?.region && options.extent) {
+    const regionDegrees = node.boundingVolume.region.map(CMath.toDegrees);
```

Comment:
Let's transform the extent to radian. That way we don't have to convert all these volumes.